inc/scripts.php

```diff
@@ -34,11 +34,24 @@ function _s_scripts() {
 	$asset_file = include( __DIR__ . '/../build/index.asset.php' );
 
 	// Register styles & scripts.
-	wp_enqueue_style( 'wd_s', get_stylesheet_directory_uri() . '/build/index.css', array(), $asset_file['version'] );
+	wp_enqueue_style( 'wd_s', get_stylesheet_directory_uri() . '/build/style-index.css', array(), $asset_file['version'] );
 	wp_enqueue_script( 'wds-scripts', get_stylesheet_directory_uri() . '/build/index.js', $asset_file['dependencies'], $asset_file['version'], true );
 
 	if ( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
 		wp_enqueue_script( 'comment-reply' );
 	}
 }
 add_action( 'wp_enqueue_scripts', '_s_scripts' );
+
+/**
+ * Enqueue editor scripts.
+ *
+ * @author WDS
+ */
+function _s_editor_scripts() {
+	$asset_file = include( __DIR__ . '/../build/index.asset.php' );
+
+	wp_enqueue_style( '_s_editor_styles', get_stylesheet_directory_uri() . '/build/index.css', [ 'wp-edit-blocks' ], $asset_file['version'] );
+}
+
+add_action( 'enqueue_block_editor_assets', '_s_editor_scripts' );
```

Comment:
@michealengland Instead of all this, could you just use the following in `functions.php`?

```php
	// Add support for editor styles.
	add_theme_support( 'editor-styles' );

	// Enqueue editor styles.
	add_editor_style( 'build/index.css' );
```

Reply:
I like the simplified hooks, I'll check it out. Thanks for the share!